src/components/Settings/integrations/Google/Modals/EditGoogleAccountModal.test.tsx

```diff
@@ -510,13 +471,12 @@ describe('EditGoogleAccountModal', () => {
           variant: 'success',
         },
       );
-      expect(mutationSpy.mock.calls[2][0].operation.operationName).toEqual(
-        'SyncGoogleAccount',
-      );
-      expect(mutationSpy.mock.calls[2][0].operation.variables.input).toEqual({
-        googleAccountId: googleAccount.id,
-        integrationName: 'calendar',
-        googleIntegrationId: googleIntegration.id,
+      expect(mutationSpy).toHaveGraphqlOperation('SyncGoogleAccount', {
```

Comment:
Thanks for taking the opportunity to switch to `toHaveGraphqlOperation` so that this test never breaks again from changing the number of GraphQL calls!